compiler/src/dotty/tools/dotc/parsing/Parsers.scala

```diff
@@ -4,23 +4,24 @@ package parsing
 
 import scala.collection.mutable.ListBuffer
 import scala.collection.immutable.BitSet
-import util.{ SourceFile, SourcePosition }
+import util.{SourceFile, SourcePosition}
 import Tokens._
 import Scanners._
 import MarkupParsers._
 import core._
 import Flags._
 import Contexts._
 import Names._
-import ast.Positioned
+import ast.{Positioned, Trees, untpd}
 import ast.Trees._
 import Decorators._
 import StdNames._
 import util.Positions._
 import Constants._
 import ScriptParsers._
 import Comments._
-import scala.annotation.{tailrec, switch}
+
+import scala.annotation.{switch, tailrec}
```

Comment:
same with this, if you intended to move this so that `scala` imports are separate from internal imports, then this should be at the top with the other one. Otherwise let's get rid off this change.